db/deploy/tables/rfi_data.sql

```diff
@@ -0,0 +1,41 @@
+-- deploy ccbc:tables/rfi_data to pg
+
+begin;
+
+create table ccbc_public.rfi_data(
```

Comment:
Do we want to create a new table for the RFI data? I was thinking we'd reuse the `form_data` table to hold the RFIs and to differentiate them based on type

Reply:
I dont think re-using `form_data` would be feasible as it would require custom RLS (one form type belongs to customer, another to analyst). And application can have multiple RFIs .. Separate table seems to be cleaner solution.